src/mobile/components/layout/templates/main.jade

```diff
@@ -16,4 +16,4 @@ block body
       != stitch && stitch.components.NavBar({ user: sd.CURRENT_USER, notificationCount: sd.NOTIFICATION_COUNT })
       block content
 
-    include footer
+    include ../../../../desktop/components/main_layout/footer/template
```

Comment:
Can we stitch the new footer in here, like we stitch in the navbar just above?

Reply:
Is there a reason to do that vs. just reusing the template? I think it'd be nice to have a single file that we use for the footer on old Force pages.

Reply:
Oh I see! Things are stitched in, but in that file 👍